### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by leolivier** [fauxauldrich/camouflage#250] [General]
Hello,
I noticed the last commit in this repo is 7 months old. 
Should I start using Camouflage for a company use case or is it more or less dead ?
(Absolutely no offense in this question, I can totally understand nobody has time to work on this for free :) but I need to know as I won't be able to maintain it myself)

**Comment by shubhendumadhukar** [comment]
Hello,

That's a fair query. No offence taken. Come to think of it, I should pin this.

To answer your question, this project is pretty much stagnant for now. I had started it as a project for use at my organisation which I have left a while back, and I don't intend to maintain it for now.

I do however have plans to continue this project differently, in a way that is easy to maintain and extend. I have started the work on it ([preview here](https://testinggospels.github.io/camouflage-docs/)), but I have not been able to make as much progress as I would have liked to, due to my other commitments. So I am afraid don't have any timelines there as well.

**Reply by leolivier** [reply]
Crystal clear!
Thanks for your honesty @shubhendumadhukar
